runtime/v2/shim/shim_windows.go

```diff
@@ -108,12 +108,11 @@ func serveListener(path string) (net.Listener, error) {
 
 func handleSignals(logger *logrus.Entry, signals chan os.Signal) error {
 	logger.Info("starting signal loop")
+
 	for {
-		select {
-		case s := <-signals:
+		for s := range signals {
 			switch s {
 			case os.Interrupt:
-				break
```

Comment:
the original one doesn't break out of an outer loop so that I remove the `break` statement here